tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/LetInExpander.scala

```diff
@@ -80,7 +80,7 @@ class LetInExpander(tracker: TransformationTracker, keepNullary: Boolean) extend
 }
 
 object LetInExpander {
-  def apply(tracker: TransformationTracker, keepNullary: Boolean)(implicit typeTag: TypeTag): LetInExpander = {
+  def apply(tracker: TransformationTracker, keepNullary: Boolean): LetInExpander = {
```

Comment:
Why does this change for VMT involve removing the implicit typetag for the let/in expander?

Reply:
Because I just happened to notice it while I was cleaning up unused imports in `InlinePass`. It's 100% unnecessary to have this implicit here.

Reply:
Ah, make sense. I thought it might be related in some way I missed.

As a note for the future: a goo way to help cut down on PR length is to make these incidental, unrelated fixes in a separate, trivial PR.